docs/about.md

```diff
@@ -127,13 +120,11 @@ We also support the [NHS Data Science Community](https://data-science-community.
     <tr><td>Ben Wallace</td><td>Senior Data Scientist</td><td>Central Data Science Team</td><td></td> </tr>
     <tr><td>Jane Kirkpatrick</td><td>Senior Data Scientist</td><td>Central Data Science Team</td><td></td> </tr>
     <tr><td>Kenneth Quan</td><td>Senior Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/quan14">quan14</a></td> </tr>
-    <tr><td>Daniel Goldwater</td><td>Senior Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/DanGoldwater1">DanGoldwater1</a></td> </tr>
     <tr><td>Shoaib Ali Ajaib</td><td>Senior Data Scientist</td><td>National SDE Team</td> <td></td></tr>
     <tr><td>Marek Salamon</td><td>Senior Data Scientist</td><td>National SDE Team</td><td></td> </tr>
     <tr><td>Adam Hollings</td><td>Senior Data Scientist</td><td>National SDE Team</td><td><a href="https://github.com/AdamHollings">AdamHollings</a></td> </tr>
     <tr><td>Oluwadamiloju Makinde</td><td>Senior Data Scientist</td><td>National SDE Team</td> <td></td></tr>
     <tr><td>Joseph Wilson</td><td>Senior Data Scientist</td><td>National SDE Team</td><td><a href="https://github.com/josephwilson8-nhs">josephwilson8-nhs</a></td> </tr>
-    <tr><td>Alistair Jones</td><td>Senior Data Scientist</td><td>National SDE Team</td><td><a href="https://github.com/alistair-jones">alistair-jones</a></td> </tr>
```

Comment:
I don't like erasing people from history - instead could we not put "former" Data scientist or something? Or have another column which highlights current vs former staff  - it's not consistent too - currently I'm no longer in the team, as with Alistair, and yet I'm still represented!